src/lib/CPAuth/config.ts

```diff
@@ -0,0 +1,96 @@
+import { getAuthorityURLFromGSApiURL } from 'lib/CPAuth/util';
+import { IOAuth2Config, IOAuth2SigningKey } from 'lib/OAuth2/OAuth2';
+import { AppRoutes } from 'shared/constants/routes';
+
+const authority = getAuthorityURLFromGSApiURL(window.config.apiEndpoint);
+
+const signingKeys: IOAuth2SigningKey[] = [
```

Comment:
Unfortunately we have to hard-code all this configuration, because the auth server doesn't allow cross-origin requests.